app/assets/stylesheets/dashboard/service_calendar.sass

```diff
@@ -1,3 +1,14 @@
+.user-view-ssr-modal
+  width: 95%
+
+.visit-before-container, .visit-day-container, .visit-after-container
```

Comment:
Sorry, I thought I responded to this yesterday. Anyways, I think it's a good idea to start moving Bootstrap classes into the stylesheets like this. After all, the view files should be devoted to markup, not styling.